quelpa.el

```diff
@@ -423,6 +429,14 @@ Passing the option is necessary on the systems where the default
 tar format isn't gnu."
   :type 'boolean)
 
+(defvar quelpa-build-bsdtar-p
+  (when quelpa-build-tar-executable
+    (string-match-p
+     "bsdtar"
+     (shell-command-to-string
```

Comment:
We should avoid doing this shell command at load time, because it will increase Emacs startup time.  Especially since, as it's a shell command, some systems (like Ubuntu) may do extra work trying to provide help to the user when the command is not found, like searching for similar command names in a package database.

Reply:
I will delay initialization of this until we first using tar.